integration/service/oasis/stationfinder/tnsearchfinder/tnsearch_station_finder.go

```diff
@@ -0,0 +1,34 @@
+package tnsearchfinder
+
+import (
+	"github.com/Telenav/osrm-backend/integration/pkg/api/nav"
+	"github.com/Telenav/osrm-backend/integration/pkg/api/oasis"
+	"github.com/Telenav/osrm-backend/integration/service/oasis/searchconnector"
+	"github.com/Telenav/osrm-backend/integration/service/oasis/stationfinder/stationfindertype"
+)
+
+type tnSearchStationFinder struct {
+	sc *searchconnector.TNSearchConnector
+}
+
+// NewTnSearchStationFinder creates finder based on telenav search
+func NewTnSearchStationFinder(sc *searchconnector.TNSearchConnector) *tnSearchStationFinder {
```

Comment:
Simply `New()`

Reply:
Copy that, package name already indicate for what to new